.github/workflows/release.yml

```diff
@@ -57,12 +72,13 @@ jobs:
               cd $DIRECTORY && npx chrome-webstore-upload-cli@2 upload --auto-publish
               ;;
             firefox)
-              cd $DIRECTORY && npx web-ext-submit@6
+              cd $DIRECTORY && npx web-ext-submit@7
```

Comment:
Hopefully the release script still works 😬 

It's been a few months since the last release